WcaOnRails/app/controllers/results_controller.rb

```diff
@@ -29,49 +29,82 @@ def rankings
     @show = splitted_show_param[0].to_i
     @is_persons = splitted_show_param[1] == "persons"
     @is_results = splitted_show_param[1] == "results"
-    limit_condition = "LIMIT #{@show}"
+    expanded_limit_condition = "LIMIT #{@show * 2}"
+
+    # Required fields gotten from LightResult
+    result_select = "
```

Comment:
I generally like that we aren't using result.* anymore, do you know what happens with the duplicate values that are generated here?

Reply:
Duplicate values?